torch/_prims_common/__init__.py

```diff
@@ -1576,6 +1576,27 @@ def mask_tensor(mask: TensorLikeType, t: TensorLikeType):
         return torch.where(mask, t, 0)
 
 
+def get_aten_op(fn: Callable, name: str):
+    """
+    Given the __module__ of reference and its name, it returns
```

Comment:
Isn't there a reference->ATen mapping that this should be using?

Reply:
I have no idea. I don't think such a thing exists. Perhaps @ezyang knows?

Reply:
Whenever a ref is registered as a decomposition for an aten op (`register_decomposition`) that also sets up a ref to aten mapping. We don't publish this API but the info is there inside a hash table.

Reply:
Yep, that's what I thought. This function is meant to be called in order to then register the relevant decomposition later on, so we can't make use of that.